docs/reference/release-notes/7.12.asciidoc

```diff
@@ -0,0 +1,298 @@
+:es-issue: https://github.com/elastic/elasticsearch/issues/
+:es-pull:  https://github.com/elastic/elasticsearch/pull/
+
```

Comment:
These are already pulled in via our [docs attributes file](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L366). We can remove them here.
```suggestion
```